### PR TITLE
Prep for 4.2.11 release

### PR DESCRIPTION
**Pull request by MGatner**
**Description**
Updates changelog and version references for 4.2.11.

**Checklist:**
- [X] Securely signed commits
- [X] Component(s) with PHPDoc blocks, only if necessary or adds value
- [X] Unit testing, with >80% coverage
- [X] User guide updated
- [X] Conforms to style guide

### CODE REVIEW

File: user_guide_src/source/installation/upgrade_4211.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
+##############################
+Upgrading from 4.2.9 to 4.2.10
```
Comment (paulbalandan) [inline user_guide_src/source/installation/upgrade_4211.rst:2]:
```suggestion
Upgrading from 4.2.10 to 4.2.11
```

File: user_guide_src/source/installation/upgrade_4211.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+- :ref:`Manual Installation Upgrading <installing-manual-upgrading>`
+
+.. contents::
+    :local:
+    :depth: 2
+
+Project Files
+*************
+
+Version ``4.2.10`` did not alter any executable code in project files.
```
Comment (paulbalandan) [inline user_guide_src/source/installation/upgrade_4211.rst:18]:
```suggestion
Version ``4.2.11`` did not alter any executable code in project files.
```

File: user_guide_src/source/installation/upgrade_4211.rst (hunk 1 above)
Comment (MGatner) [reply user_guide_src/source/installation/upgrade_4211.rst:2]: Thanks Paul

File: user_guide_src/source/installation/upgrade_4211.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
+##############################
+Upgrading from 4.2.10 to 4.2.11
+##############################
```
Comment (paulbalandan) [inline user_guide_src/source/installation/upgrade_4211.rst:3]: I'm not sure if this needs updating of the over and under lines?
```suggestion
###############################
Upgrading from 4.2.10 to 4.2.11
###############################
```

Comment (paulbalandan) [reply user_guide_src/source/installation/upgrade_4211.rst:3]: My view from the browser tells me it needs updating. Not sure from local view.

Comment (kenjis) [inline user_guide_src/source/installation/upgrade_4211.rst:3]:
```suggestion
###############################
Upgrading from 4.2.10 to 4.2.11
###############################
```

Comment (kenjis) [reply user_guide_src/source/installation/upgrade_4211.rst:3]: It needs.
> user_guide_src/source/installation/upgrade_4211.rst:1: WARNING: Title overline too short.

Comment (MGatner) [reply user_guide_src/source/installation/upgrade_4211.rst:3]: Probably?

**Pull request merged**